packages/database/lib/modular/index.js

```diff
@@ -98,18 +98,18 @@ export function forceWebSockets() {
 }
 
 /**
- * @param {Database} db
- * @returns {Date}
+ * @returns {object}
  */
-export function getServerTime(db) {
-  return db.getServerTime();
+export function serverTimestamp() {
+  return firebase.database.ServerValue.TIMESTAMP;
 }
 
 /**
- * @returns {object}
+ * @param {Database} db
+ * @returns {Date}
  */
-export function serverTimestamp() {
-  return firebase.database.ServerValue.TIMESTAMP;
+export function getServerTime(db) {
```

Comment:
why two functions instead of having the first one check if db === undefined, followed by each branch doing the right thing? That idiom is already in a lot of the code, for what it's worth